--- configure.orig	2019-12-01 13:31:58.000000000 -0500
+++ configure	2022-12-01 13:32:25.000000000 -0500
@@ -325,7 +325,7 @@
 ;;
 
   apple-osx-gcc)   # Macintosh computer running OSX, using the Gnu C compiler.
-    CC=gcc
+    CC=$CCOMPL
     FC=gfortran
     CFLAGS="$CFLAGS -Dapple_osx"
 #
@@ -346,7 +346,7 @@
 ;;
 
   intel-osx-gcc)   # Macintosh computer running OSX, using the Gnu C compiler.
-    CC=gcc
+    CC=$CCOMPL
     FC=gfortran
     CFLAGS="$CFLAGS -Dintel_osx"
 #
